src/map/mob.cpp

```diff
@@ -6107,10 +6107,15 @@ static bool mob_parse_row_mobskilldb( char** str, size_t columns, size_t current
 		ms->val[1] = 0; //Do not "set" it.
 	}
 
-	if(*str[17])
-		ms->emotion = atoi(str[17]);
+	if(*str[17]){
+		int temp_emotion = atoi(str[17]);
+		if(temp_emotion > ET_BLANK && temp_emotion < ET_MAX)
```

Comment:
actually have monsters on RE db using 2000+ emotiontype, it prevents any send of unknow  emotiontype